cfn-resources/integration-tests/util/atlasHelper.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"runtime"
+
+	"github.com/mongodb-forks/digest"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/version"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+	realmAuth "go.mongodb.org/realm/auth"
+	"go.mongodb.org/realm/realm"
+)
+
+type AtlasEnvOptions struct {
+	OrgID      string
+	PrivateKey string
+	PublicKey  string
+	BaseURL    string
+}
+
+const (
+	cfnTool = "mongodbatlas-cloudformation-resources"
+)
+
+var (
+	toolName           = cfnTool
+	userAgent          = fmt.Sprintf("%s/%s (%s;%s)", toolName, version.Version, runtime.GOOS, runtime.GOARCH)
+	terraformUserAgent = "terraform-provider-mongodbatlas"
+)
+
+func GetNewAtlasTeam(ctx context.Context, client *atlas.Client, name string, orgID string) (*atlas.Team, error) {
```

Comment:
[it's neither idiomatic nor necessary to put Get into a getter's name](https://go.dev/doc/effective_go#Getters)

Reply:
also this is more of a creator than a getter

Reply:
same comment for all `Get*` in this PR

Reply:
updated